docs/DeveloperGuide.md

```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Run jar file in command prompt: Expected: Shows the GUI with a set of sample data. The window size may not be optimum.
 
 1. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
```

Comment:
Can change to `Re-launch the app by running on the command prompt and enter  'java -jar wanderlust.jar`.